app/globals.css

```diff
@@ -178,10 +178,12 @@
   [class*="logo-letter-animated"] {
     --delay-step-increment: 0.25s;
     --animation-length: 3s;
-    animation: wave-y-5 var(--animation-length) ease-in-out infinite;
     /* Special more gentle animation for users with prefers-reduced-motion set */
-    @media (prefers-reduced-motion) {
-      animation: wave-y-3 var(--animation-length) ease-in-out infinite;
```

Comment:
Flipped this around. wave-y-3 is now the default, and wave-y-5 is the motion-safe variant.

motion-safe in Tailwind = prefers-reduced-motion: no-preference in CSS